content/courses/connecting-to-offchain-data/oracles.md

```diff
@@ -4,7 +4,7 @@ objectives:
   - Explain why onchain programs cannot readily access real-world data on their
     own
   - Explain how oracles solve the problem of accessing real-world data onchain
-  - Explain how incentivized oracle networks make data more trustworthy
+  - Explain how incentivized Oracle networks make data more trustworthy
```

Comment:
oracle is lowercase.

Reply:
I've made the change throughout the article